tests/navbar.test.ts

```diff
@@ -0,0 +1,85 @@
+import { expect, test, type Page } from '@playwright/test';
+
+//username testschnatter
+//password Testtest123!
+
+async function login(page: Page) {
```

Comment:
kann man das eventuell irgendwie auslagern, dass es die funktion nur einmal gibt

Reply:
liegt jz in testMethods.ts